core/src/main/java/com/datastrato/gravitino/Configs.java

```diff
@@ -45,6 +49,13 @@ public interface Configs {
           .stringConf()
           .createWithDefault(DEFAULT_ENTITY_KV_STORE);
 
+  ConfigEntry<String> ENTITY_RELATION_STORE =
+      new ConfigBuilder(ENTITY_RELATION_STORE_KEY)
+          .doc("Detailed implementation of relation storage")
+          .version("0.5.0")
+          .stringConf()
```

Comment:
Could you use method `checkValue` to check the config option?

Reply:
done

Reply:
done